zone/spells.cpp

```diff
@@ -2736,12 +2736,14 @@ bool Mob::ApplyBardPulse(int32 spell_id, Mob *spell_target, CastingSlot slot) {
 		Live does not spam client with do not take hold messages. Checking here avoids that from happening. Only try to reapply if charm fades.
 	*/
 	if (spell_target->IsCharmed() && spells[spell_id].mana == 0 && spell_target->GetOwner() == this && IsEffectInSpell(spell_id, SE_Charm)) {
+        if (IsClient()) CastToClient()->CheckSongSkillIncrease(spell_id);
```

Comment:
Indenting type.

Also formatting style. Please avoid single line implied entries.

e.g. replace with:

```cpp
		if (IsClient()) { 
			CastToClient()->CheckSongSkillIncrease(spell_id);
		}
```

This helps with consistency in formatting across the code base and increases readability.

Reply:
Woops sorry my IDE not all set up correctly yet